packages/flutter_tools/lib/src/linux/build_linux.dart

```diff
@@ -56,9 +56,11 @@ Future<void> buildLinux(
   environmentConfig['FLUTTER_TARGET'] = target;
   final LocalEngineInfo? localEngineInfo = globals.artifacts?.localEngineInfo;
   if (localEngineInfo != null) {
-    final String engineOutPath = localEngineInfo.engineOutPath;
+    final String engineOutPath = localEngineInfo.targetOutPath;
```

Comment:
Rename this local to targetOutPath for consistency.

Reply:
Done!